drivers/usb/udc/udc_stm32.c

```diff
@@ -211,15 +211,15 @@ void HAL_PCD_ResetCallback(PCD_HandleTypeDef *hpcd)
 
 	/* Re-Enable control endpoints */
 	ep = udc_get_ep_cfg(dev, USB_CONTROL_EP_OUT);
-	if (ep && ep->stat.enabled) {
+	if ((ep != NULL) && ep->stat.enabled) {
```

Comment:
It would be better to rename `struct udc_ep_config *ep;` to struct `udc_ep_config *ep_cfg;` or `struct udc_ep_config *cfg` because ep is usually used for endpoint address.

Then I am fine with `if (ep_cfg != NULL && ep->stat.enabled) {` but not with unnecessary parentheses like `if ((ep != NULL) && ep->stat.enabled) {`.

Reply:
OK.